src/main/java/io/featureflow/client/model/Event.java

```diff
@@ -39,6 +39,16 @@ public Event(String goalKey, FeatureflowUser user, Map<String, String> evaluated
         this.evaluatedVariants = evaluatedVariants;
     }
 
+    @Override
+    public String toString() {
+        return "Event{" +
+                "user=" + user.getId() +
+                ", featureKey='" + featureKey + '\'' +
+                ", goalKey='" + goalKey + '\'' +
+                ", type='" + type + '\'' +
+                '}';
+    }
+
```

Comment:
Added a basic toString() here which is used in debug logging, you can add/remove fields if that makes more sense